### ISSUE TITLE
Adapt to allow for ReadOnlyRootFilesystem

### ISSUE THREAD

**Issue by Ornias1993** [Mozart409/blocky-frontend#12]
On deployment platforms such as kubernetes, an often deployed extra security precaution is "readOnlyRootFilesystem".
This basically means that unless storage/volumes are mounted an App cannot write anywhere.

As this is just a basic UI, it should be possible to adapt it in such a way that the writes are limited to specific folders that don't depend on content being present at start (aka: can be empty at start).

**Comment by Mozart409** [comment]
Yes u are right again. Do you have any examples how to do that or can you open a pr?

Thank you

**Comment by simonmcnair** [comment]
I don't see anywhere that this actually writes to the fs, although I'm not really a ts/react developer.  I can only see env variables, and no logging.

Where are writes being written ?

**Comment by Mozart409** [comment]
There are no writes to the file system and I don't know how this could be fixed.
